pkg/process/procutil/process_linux_test.go

```diff
@@ -619,21 +599,38 @@ func testParseIO(t *testing.T) {
 	defer probe.Close()
 
 	pids, err := probe.getActivePIDs()
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	for _, pid := range pids {
 		actual := probe.parseIO(filepath.Join(probe.procRootLoc, strconv.Itoa(int(pid))))
 		expProc, err := process.NewProcess(pid)
-		assert.NoError(t, err)
+		require.NoError(t, err)
 		expIO, err := expProc.IOCounters()
-		assert.NoError(t, err)
-		assert.Equal(t, expIO.ReadCount, actual.ReadCount)
-		assert.Equal(t, expIO.ReadBytes, actual.ReadBytes)
-		assert.Equal(t, expIO.WriteCount, actual.WriteCount)
-		assert.Equal(t, expIO.WriteBytes, actual.WriteBytes)
+		require.NoError(t, err)
+		assert.Equal(t, int64(expIO.ReadCount), actual.ReadCount)
+		assert.Equal(t, int64(expIO.ReadBytes), actual.ReadBytes)
+		assert.Equal(t, int64(expIO.WriteCount), actual.WriteCount)
+		assert.Equal(t, int64(expIO.WriteBytes), actual.WriteBytes)
```

Comment:
Could we use `ConvertFromIOStats` here and compare the the `IOCountersStat` structs?

Reply:
Ah yea 😆  since it's a test I skipped that. I can do it.